Src/Toasts.swift

```diff
@@ -0,0 +1,218 @@
+//
+//  Toasts.swift
+//  project
+//
+//  Created by Gustaf Gunér on 2019-07-11.
+//
+
+import Foundation
+import Flow
+import Form
+import UIKit
+
+enum ToastHideDirection {
+    case left
+    case right
+}
+
+enum ToastSymbol {
```

Comment:
You could probably use an `Either<ImageAsset, Character>` instead

Reply:
Neat!